test_scripts/Defects/5_0/2379_SDL_must_send_log_error_and_ignore_invalid_RPC_in_case_SDL_cuts_off_fake_parameters_and_RPC_becomes_invalid.lua

```diff
@@ -0,0 +1,90 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2379
+--
+-- Precondition:
+-- SDL must send log error and ignore invalid RPC in case SDL cuts off fake parameters and RPC becomes invalid
+-- Description:
+-- Steps to reproduce:
+-- 1) SDL cuts off fake parameters from response (request) that SDL should use internally and this response (request) is invalid by any reason
+-- Expected:
+-- 1) Log the corresponding error
+-- 2) Ignore this response (request)
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local variables ]]
+local requestParamsValidData = {
```

Comment:
the better idea to check as valid case the same request as invalid but with valid type of menuName, to make sure that sdl process request with fake parameter as valid. 
Also in case request will contain fake parameter the ui request must be checked for absence of this parameter.

Reply:
Fixed in commit: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2006/commits/928a367326254768c208ca9471e42b12269baf52

Reply:
Changes after comments for PR: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2006/commits/928a367326254768c208ca9471e42b12269baf52

Reply:
Request is not checked for absence of fake parameter.